datasets/si/br/crawler.py

```diff
@@ -0,0 +1,86 @@
+
+    response = context.fetch_response(page_url, headers=HEADERS)
+    return response.json()
+
+
+def create_entities(context: Context, record: Dict[str, Any]):
+    organization = context.make("Organization")
+
+    organization_name = record.get("org_naziv")
+    organization_number = record.get("org_sifrapu")
+
+    organization.id = context.make_id(organization_number or organization_name)
+    organization.add("name", organization_name)
+    organization.add("idNumber", organization_number)
+
+    legal_entity = context.make("LegalEntity")
+    subject_name = record.get("subjekt_naziv")
+    registration_number = record.get("subjekt_maticna")
+
+    if not (subject_name and registration_number):
+        context.log.error("Subject name and registration number not found")
+        return
+
+    legal_entity.id = context.make_id(registration_number or subject_name)
+    legal_entity.add("name", subject_name)
+    legal_entity.add("registrationNumber", registration_number)
+    legal_entity.add("taxNumber", record.get("subjekt_davcna"))
+
+    if registration_number:
+        legal_entity.add(
+            "sourceUrl",
+            f"https://erar.si/transakcija/placnik/{organization_number}/prejemnik/{registration_number}",
+        )
+
+    validity_link = context.make("UnknownLink")
+    validity_link.id = context.make_id(legal_entity.id)
+    validity_link.add("startDate", record.get("od"))
+
+    if not record.get("do").startswith("9999"):
+        validity_link.add("endDate", record.get("do"))
+
+    validity_link.add(
+        "description",
+        "When the restriction is valid from, and when it ends. No end date implies it is valid until canceled",
+    )
+    validity_link.add("subject", legal_entity)
+    validity_link.add("object", organization)
```

Comment:
totally see why you added the org and the link here.
in this case let's only do the legal entity, and the debarment topic, and note why these entities are debarred in the entity program property.